### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by spond**
Fixing the segfault issue

### CODE REVIEW

**Comment by stevenweaver** [comment]
Hi Sergei, 

I'm going to put in place a unit test before accepting this PR, so it may take until Monday for acceptance. 

Best,
Steven

**Comment by spond** [comment]
Dear @stevenweaver,

No problem. Not critical, as an edge case is being addressed.

Best,
Sergei

File: src/TN93.cpp [hunk 1]
```diff
@@ -323,14 +323,15 @@ int main(int argc, const char *argv[]) {
   */
 
   bool cross_comparison_only = (args.input2 && !do_fst);
+  unsigned long system_CPU_count = omp_get_max_threads();
```
Comment (spond) [inline src/TN93.cpp]: You can actually delete this line, because `system_CPU_count` is left over from my debugging statements.
